plugins/backend/wiz-backend/src/service/router.ts

```diff
@@ -34,10 +34,34 @@ export async function createRouter(
   const router = Router();
   const wizAuthClient = new WizClient(config);
 
-  await wizAuthClient.fetchAccessToken();
-
   router.get('/wiz-issues/:projectId', async (req, res) => {
     try {
+      const wizConfig = {
```

Comment:
Move this to be a catch-all router instead, otherwise we'd need to do all these checks on all routes